### PR TITLE
move cypress-json

### PR DESCRIPTION
**Pull request by itailiors**

### CODE REVIEW

**Comment by itailiors** [comment]
updated the GitHub Actions workflow to specify the path to the Cypress configuration file, 
and moved cypress.config.js

**Comment by dangershony** [comment]
Is this ready to review?

**Comment by itailiors** [comment]
> Is this ready to review?

Technically yes, I can add more cypress tests and optimize it in another branch if you want to review this one 

If so i'll tell you generally what there is here:
updated the GitHub Actions workflow to specify the path to the Cypress configuration file,
and moved cypress.config.js
Updated CONTRIBUTING.MD file to match the change (now you have to run cypress like that command there if not then cypress will think its not configured)
Added new cypress testing spec for browse tab
refactored cypress commands.
and test fix (there is a flaky line that sometimes works sometimes doesn't, so its commented atm).

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -65,4 +65,5 @@ jobs:
           build: npm run build
           start: npm start
           browser: electron
-          headless: true      
+          headless: true
+          config-file: src/Testing/cypress.config.js
```
Comment (dangershony) [inline .github/workflows/pull-request.yml:69]: That was a pretty easy change actually good job

**Review by dangershony — DISMISSED** [review]

**Comment by dangershony** [comment]
You may need to fix conflicts before I commit

**Comment by itailiors** [comment]
> You may need to fix conflicts before I commit

Fixed.

**Review by dangershony — APPROVED** [review]

**Pull request merged**